src/main/kotlin/org/rust/ide/statistics/RsConvertJsonToStructUsagesCollector.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.statistics
+
+import com.intellij.internal.statistic.eventLog.EventLogGroup
+import com.intellij.internal.statistic.eventLog.events.EventFields
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+
+@Suppress("UnstableApiUsage")
+object RsConvertJsonToStructUsagesCollector : CounterUsagesCollector() {
+
+    override fun getGroup(): EventLogGroup = GROUP
+
+    private val GROUP = EventLogGroup("rust.generate.type.from.json.usage", 1)
+
+    private val JSON_LIKE_TEXT_PASTED = GROUP.registerEvent("json.like.text.pasted")
+    private val PASTED_JSON_CONVERTED = GROUP.registerEvent("pasted.json.converted")
+
+    private val REMEMBER_CHOICE_RESULT = GROUP.registerEvent(
+        "remember.choice.result",
```

Comment:
nit: maybe introduce a common prefix for these events to simplify future search?

Reply:
I guess "json.like.text.pasted" and "pasted.json.converted" are unique enough, will it be ok to replace just "remember.choice.result" to "json.paste.dialog.remember.choice.result"?